### PR TITLE
#1129 Issue Fix

### LINKED ISSUE #1129: taxanomy option just work in edit mode ?
Hi, 
Follow instruction, i create category option and place it on my theme where:
\framework\theme\options\taxonomies\category.php

Problem is: when i create new category from : Blog Post / Category.. I can see category option. But when edit it.
Question is: how to enable taxanomy option when we create new it. [now, we just see it when we do edit any already category for example]

Please Advice
Thanks in Advances
Cuong

### PR DESCRIPTION
**Pull request by betuha**
#1129 Issue fix added; taxonomy options will be added on taxonomy term creating

### CODE REVIEW

File: framework/core/components/backend.php [hunk 1]
```diff
@@ -747,6 +747,37 @@ public function _action_create_taxonomy_options( $term ) {
...
+			'limit_container_types' => array(), // only simple options are allowed on taxonomy edit page
+			'limit_level'           => 1,
+		) );
+
+		if ( empty( $collected ) ) {
+			return;
+		}
+		$values = array();
+		foreach ( $options as $option_id => $values ) {
+			$values[ $option_id ] = $values['value'];
```
Comment (moldcraft) [inline framework/core/components/backend.php]: This is non sense. On options render default values will be extracted automatically

File: framework/core/components/backend.php [hunk 2]
```diff
@@ -1018,6 +1055,19 @@ public function _sync_post_separate_meta( $post_id ) {
...
+	 * @param int $term_id
+	 */
+	public function _action_save_taxonomy_fields( $term_id ) {
+		if ( ! isset( $_POST['action'] ) || ! isset( $_POST['taxonomy'] ) ) {
+			return; // this is not real term form submit, abort save
+		}
+		$taxonomy = $_POST['taxonomy'];
+		$custom   = $_POST['fw_options'];
+		foreach ( $custom as $id => $value ) {
+			fw_set_db_term_option( $term_id, $taxonomy, $id, $value );
```
Comment (moldcraft) [inline framework/core/components/backend.php]: Very bad/insecure way to save valued in db, you must use `fw_get_options_values_from_input()`

**Comment by moldcraft** [comment]
Thank you. I will inspect and accept it soon.

**Comment by betuha** [comment]
I have accepted your pull request

**Comment by moldcraft** [comment]
Thank you

**Pull request merged**